### PR TITLE
inference for MoleculeNet fine-tuned models BBBP, TOXICITY, FDA_APPR

### PR DESCRIPTION
**Pull request by simona-rc**

### CODE REVIEW

File: mammal/examples/molnet/molnet_infer.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+def main(task_name: str, smiles_seq: str, device: str):
+    task_dict = load_model(task_name=task_name, device=device)
+    result = task_infer(task_dict=task_dict, smiles_seq=smiles_seq)
+    print(f"The prediction for {smiles_seq=} is {result}")
+
+
+def load_model(task_name: str, device: str) -> dict:
+    match task_name:
+        case "BBBP":
+            path = "SagiPolaczek/test4Simona"  # "ibm/biomed.omics.bl.sm.ma-ted-458m.moleculenet_bbbp"
```
Comment (mosheraboh) [inline mammal/examples/molnet/molnet_infer.py:36]: Why is it still under Sagi?

**Review by mosheraboh — APPROVED** [review]
Looks great!
Can you please also create a simple unit-test that runs this example?

File: mammal/examples/molnet/molnet_infer.py (hunk 1 above)
Comment (simona-rc) [reply mammal/examples/molnet/molnet_infer.py:36]: Moved to ibm after we got the approval

Comment (simona-rc) [reply mammal/examples/molnet/molnet_infer.py:36]: Also, added a simple unit-test

**Pull request merged**